CHANGELOG.md

```diff
@@ -1,3 +1,69 @@
+CHANGES IN VERSION 1.22.0 (FROM 1.21.0)
+===================================
+
+Appium 1.22.0 is a minor release
+
+This release is the **finial** release for Appium 1.x seriese.
+We're going to move to Appium 2.x [TODO: link to documentation].
+Appium 1.x will have patch releases, but it will no longer get minor releases.
+
+### General
+
+* Add `imageMatchMethod` to be able to customize image mathcing method in [find element by image](http://appium.io/docs/en/advanced-concepts/image-elements/) [appium-base-driver#480](https://github.com/appium/appium-base-driver/pull/480) [appium-support#222](https://github.com/appium/appium-support/pull/222)
+* `appium:enableWebviewDetailsCollection` capability defaults to `true` [appium-android-driver#699](https://github.com/appium/appium-android-driver/pull/699) (Thanks @mwakizaka)
+* Support `satellites` option in `setGeoLocation` command as [geo command on emulators](https://developer.android.com/studio/run/emulator-console) [appium-adb#571](https://github.com/appium/appium-adb/pull/571)
+
+### Android General
```

Comment:
it also makes sense to mention the signing optimization: https://github.com/appium/appium-adb/pull/572